assets/out

```diff
@@ -57,21 +54,13 @@ fi
 
 cd $source/$resource
 
-for i in $(seq 0 $(expr "$files_count" - 1)); do
-  file=$(jq -r ".params.remove_files[$i]" < $payload)
-  osc rm -f "$file"
-done
+osc addremove
 
 if [ ! -z "$build" ]; then
   osc build --trust-all-projects $repository $arch
 fi
 
 if [ ! -z "$commit_message" ]; then
-  files_count=$(jq -r '.params.commit.add_files | length' < $payload)
-  for i in $(seq 0 $(expr "$files_count" - 1)); do
-    file=$(jq -r ".params.commit.add_files[$i]" < $payload)
-    osc add "$file"
-  done
   osc commit -m"$commit_message"
```

Comment:
No, only sources are committed to OBS. So running `osc addremove` will only add the sources.